packages/flutter/test/material/material_test.dart

```diff
@@ -208,6 +216,77 @@ void main() {
     expect(modelE.shadowColor, equals(const Color(0xFFFF0000)));
   });
 
+  group('Elevation Overlay', () {
+
+    testWidgets('applyElevationOverlay set to false does not change surface color', (WidgetTester tester) async {
+      const Color surfaceColor = Color(0xFF121212);
+      await tester.pumpWidget(Theme(
+          data: ThemeData(
+            applyElevationOverlay: false,
+            colorScheme: const ColorScheme.dark().copyWith(surface: surfaceColor),
+          ),
+          child: buildMaterial(color: surfaceColor, elevation: 8.0))
+      );
+      final RenderPhysicalShape model = getShadow(tester);
```

Comment:
Is there a reason the helper function is called `getShadow`? I was a little confused by this